src/app/Types.elm

```diff
@@ -51,3 +53,4 @@ type Msg
     | SetCup CupOfDice
     | NextServerState ServerState
     | RoundFinished RoundResult
+    | TableMsg Table.Msg
```

Comment:
Well, they need to go *somewhere*: has to be where the top is... these messages are what the main update function uses, so it has to go there. Whenever the main update function is changed, this can be changed. They're decoupled, so it doesn't bother me where it goes, but it has to go here for now the way things are set up in the main app.